packages/interop-tests/src/__test__/interop.test.ts

```diff
@@ -193,16 +173,6 @@ it('server wallet creates channel + cooperates with browser wallet to fund chann
     },
     'dummyDomain'
   );
-
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const postFundA = await fromEvent<SingleChannelOutput>(serverWallet as any, 'channelUpdated')
-    .pipe(first(containsPostfundState))
-    .toPromise();
-
-  serverWallet.on('channelUpdated', e => console.log(JSON.stringify(e)));
-  await browserWallet.pushMessage(serverMessageToBrowserMessage(await postFundA), 'dummyDomain');
-
-  await objectiveSuccededPromise;
```

Comment:
What assertion should be made at the end of this test?

Reply:
Right now this test won't work since it's using `ganache` and just an `engine`.

 I'm going to see if I can quickly update it to work with a `wallet` . If not I'll revert my changes to this test and just skip it for now (and create an issue).

Reply:
While I don't think it's going to be too much effort it seems like too much to do it in the scope of this PR. I've just skipped the test for now in a58319e

Reply:
👍 Let's keep this PR as small as possible!